python/ray/tests/test_grpc_client_credentials.py

```diff
@@ -0,0 +1,34 @@
+import pytest
```

Comment:
This test looks great! You’ll have to add it to “BUILD” for it to run

Reply:
Thank you, I totally missed it.